cartography/intel/aws/rds.py

```diff
@@ -13,6 +14,136 @@
 logger = logging.getLogger(__name__)
 
 
+@timeit
+def _dict_value_to_str(obj: Dict, key: str) -> Optional[str]:
+    """
+    Convert the value referenced by the key in the dict to a string, if it exists, and return it. If it doesn't exist,
+    return None.
+    """
+    value = obj.get(key)
+    if value is not None:
+        return str(value)
+    else:
+        return None
+
+
+@timeit
+@aws_handle_regions
+def get_rds_cluster_data(boto3_session: boto3.session.Session, region: str) -> List[Any]:
+    """
+    Create an RDS boto3 client and grab all the DBClusters.
+    """
+    client = boto3_session.client('rds', region_name=region)
+    paginator = client.get_paginator('describe_db_clusters')
+    instances: List[Any] = []
+    for page in paginator.paginate():
+        instances.extend(page['DBClusters'])
+
+    return instances
+
+
+@timeit
+def load_rds_clusters(
+    neo4j_session: neo4j.Session, data: Dict, region: str, current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    """
+    Ingest the RDS clusters to neo4j and link them to necessary nodes.
+    """
+    ingest_rds_cluster = """
+    MERGE (cluster:RDSCluster{id: {DBClusterArn}})
```

Comment:
I hadn't used UNWIND since the rest of the module didn't and thought I'd followup with a separate PR that switched the entire module to unwind. Should I do that, or add unwind for this one spot?

Reply:
Nevermind, it didn't take me too long, and I switched the entire module to unwind.